### DISCUSSION TITLE
How to run the Che-Theia IDE within Che

### DISCUSSION THREAD

**Discussion by sboudreaux96** [eclipse-theia/theia#9470] [Q&A] [answered]
Hello - I am new to Che-Theia and have gotten Che to run locally on my computer. I am able to create a workspace from the che-theia-all dev file, but cannot figure out how to run the UI through Che. It seems that there are many containers that I can use but I am not sure which one to use, and the ones I use seem to not work at all when I use yarn. Is there any place I can be directed to for help on this?

**Comment by vince-fugnitto** [comment, marked as answer]
@sboudreaux96 thank you for the discussion, please ask `che` related questions on their appropriate repo, this repo is for the eclipse theia framework.

- https://github.com/eclipse-che/che-theia

**Reply by vince-fugnitto** [reply]
cc @benoitf
